docs/review-template.html

```diff
@@ -45,11 +45,11 @@
 </style>
 </head>
 <body>
-<h1>________ and the Standard for Public Code version <span class="standard-version">0.8.0</span></h1>
+<h1>________ and the Standard for Public Code version <span class="standard-version">ba42e7f</span></h1>
```

Comment:
As this was done by hand, the version should probably be empty for this commit, as this version is not yet tied to any tag. The automation will regenerate it with the tag's version when it runs.

We could change the automation to detect the by-hand/no-tag case.